### PR TITLE
cmd/wasm-client: bump Loop to latest version

### PR DESCRIPTION
**Pull request by guggero**
With this commit we bump Loop to the latest untagged version which
includes lightninglabs/loop#500 that adds the last_hop and
outgoing_chan_set fields to the list swaps RPC.

### CODE REVIEW

**Comment by guggero** [comment]
cc @dstadulis (part of LiT update I mentioned).

File: cmd/wasm-client/go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/faraday v0.2.7-alpha.0.20220503144421-cd1e56982f09
 	github.com/lightninglabs/lightning-node-connect v0.1.9-alpha.0.20220602120524-e9964c685b18
-	github.com/lightninglabs/loop v0.18.0-beta
+	github.com/lightninglabs/loop v0.18.0-beta.0.20220602162617-992538eed917
```
Comment (Roasbeef) [inline cmd/wasm-client/go.mod:9]: Should we also do a "pretty tag" or is the next tag for loop not quite scheduled yet?

**Review by Roasbeef — APPROVED** [review]
LGTM 🐊

We should do a tag after this lands, so we can update LiT to prep for the release of 0.15.

**Comment by guggero** [comment]
Replaced by https://github.com/lightninglabs/lightning-node-connect/pull/44.

**Pull request closed without merging**